doc/vim-rspec.txt

```diff
@@ -0,0 +1,106 @@
+*vim-rspec.txt* run RSpec from within vim
+
+INTRODUCTION                                    *vim-rspec*
+
+A lightweight RSpec runner for Vim and MacVim.
+
+Repo:    https://github.com/thoughtbot/vim-rspec
+
+FUNCTIONS                                       *vim-rspec-functions*
+
+vim-rspec provides four functions for executing your specs:
+
+                                                *all-specs* *RunAllSpecs()*
+:RunAllSpecs()          Run all spec files in the project.
+                        Equivalent to running
+
+                            rspec spec
+
+                                                *current-spec-file* *RunCurrentSpecFile()*
+:RunCurrentSpecFile()   If the current file is a spec file, all the examples in
+                        the current file are run. Equivalent to running
+
+                            rspec spec/my_spec.rb
+
+                        If it's not a spec file then the |last-spec| is run again.
+
+                                                *nearest-spec* *RunNearestSpec()*
+:RunNearestSpec()       If the current file is a spec file, and the cursor is
+                        on or within an example (e.g. an it block) then only
+                        that example is run. Otherwise the example group
```

Comment:
Cool - if that's the only change, should I force-push or add a new commit? Never quite sure what the etiquette is with open source PRs